app/interfaces/api/v2/case_workers/claim.rb

```diff
@@ -52,19 +52,19 @@ def value_band_id
             end
 
             def current_claims
-              current_user.claims.caseworker_dashboard_under_assessment.search(
```

Comment:
`caseworker_dashboard_under_assessment` scope is exactly the same same as the one used by  the search method (state in Claims::StateMachine::CASEWORKER_DASHBOARD_UNDER_ASSESSMENT_STATES)